ayon_server/addons/models.py

```diff
@@ -0,0 +1,32 @@
+from typing import Literal
+
+from ayon_server.types import Field, OPModel
+
+
+class ClientSourceInfo(OPModel):
+    type: Literal["filesystem", "server", "http"] = Field(...)
+
+
+class PathDefinition(OPModel):
+    plaform: str
+    path: str
+
+
+class FilesystemSourceInfo(ClientSourceInfo):
+    type: Literal["filesystem"] = Field("filesystem")
+    paths: list[PathDefinition] = Field(default_factory=list)
```

Comment:
I realized this morning that this actually does't make sence to have it as list.
```suggestion
class PathDefinition(OPModel):
    windows: str
    linux: str
    darwin: str


class FilesystemSourceInfo(ClientSourceInfo):
    type: Literal["filesystem"] = Field("filesystem")
    path: PathDefinition = Field(default_factory=PathDefinition)
```